requirements.txt

```diff
@@ -3,7 +3,7 @@ decorator==4.0.9
 dict==0.0.4
 docopt==0.6.2
 get==0.0.4
-jsonschema>=3.0.1
+jsonschema==3.0.0a3
```

Comment:
downgrading?

Reply:
That's right. Actually, before it was some very old version (like 2.x), which was bugged and raised validation errors even when there were none. So I updated it to 3.0.1, the latest version at the time. But now I found out that this conflicts with the `jsonschema` version which OpenTargets internal validator uses. I figured it would be a good idea to use exactly the same version as they are, because at least this way we shouldn't get different validation results